### PR TITLE
Bug/fast login

### PR DESCRIPTION
**Pull request by Pilkas-git**

### CODE REVIEW

File: Moodis/Moodis/Feature/Login/SignIn/SignInForm.cs [hunk 1]
```diff
@@ -58,8 +58,13 @@ private void ButtonRegister_Click(object sender, EventArgs e)
         private void LabelSignIn_Click(object sender, EventArgs e)
         {
             CameraForm cameraForm = new CameraForm();
+            cameraForm.StartPosition = FormStartPosition.Manual;
+            cameraForm.Location = Location;
+            var registerView = new RegisterViewModel();
```
Comment (tmaciulis22) [inline Moodis/Moodis/Feature/Login/SignIn/SignInForm.cs]: `registerView` -> `registerViewModel`

File: Moodis/Moodis/Feature/Login/SignIn/SignInForm.cs [hunk 2]
```diff
@@ -58,8 +58,13 @@ private void ButtonRegister_Click(object sender, EventArgs e)
         private void LabelSignIn_Click(object sender, EventArgs e)
         {
             CameraForm cameraForm = new CameraForm();
```
Comment (tmaciulis22) [inline Moodis/Moodis/Feature/Login/SignIn/SignInForm.cs]: var could be used here

**Review by tmaciulis22 — APPROVED** [review]
Minor changes

**Review by tmaciulis22 — APPROVED** [review]

**Review by AArlauskas — APPROVED** [review]
Good fix

**Pull request merged**